api/utils/request_data_validators.py

```diff
@@ -0,0 +1,6 @@
+def validate_add_user_data(data_dict):
```

Comment:
could you rename this `api/utils/request_data_validators.py `
to `api/utils/validators.py `

Reply:
That makes sense